package.json

```diff
@@ -83,9 +83,9 @@
     "start:rinkeby": "npm start",
     "start:staging": "REACT_APP_ENS_REGISTRY_ADDRESS=0xfe03625ea880a8cba336f9b5ad6e15b0a3b5a939 npm start",
     "build": "scripts/build",
-    "build:mainnet": "REACT_APP_ETH_NETWORK_TYPE=main npm run build",
+    "build:mainnet": "REACT_APP_ETH_NETWORK_TYPE=main ARAGON_DEMO_DAO=genesis.aragonid.eth npm run build",
     "build:mainnet-infura": "REACT_APP_DEFAULT_ETH_NODE=wss://mainnet.infura.io/_ws npm run build:mainnet",
-    "build:rinkeby": "npm run build",
+    "build:rinkeby": "ARAGON_DEMO_DAO=genesis.aragonid.eth npm run build",
```

Comment:
Actually you need the provide the address for this env var.

Reply:
RIP

Reply:
Yeah... it should be an easy change to use the org name later though, I think.